zerver/views/streams.py

```diff
@@ -371,7 +371,7 @@ def add_subscriptions_backend(
                 stream_msg = "the following streams: %s" % (stream_strs,)
             else:
                 stream_msg = "a new stream #**%s**." % created_streams[0].name
-            msg = ("%s just created %s" % (user_profile.full_name, stream_msg))
+            msg = ("@**%s** just created %s" % (user_profile.full_name, stream_msg))
```

Comment:
So, this part of the change I think we may not want to do immediately, because the mention here is in a stream and thus will trigger a mention/desktop notification for the person who created it.  

Whereas the others are in PMs and thus don't have that issue.